src/docker-object-model.ts

```diff
@@ -0,0 +1,33 @@
+import { ToolRunner, IExecOptions, IExecSyncResult } from "./utilities/tool-runner";
+
+export class DockerExec{
+    private dockerPath: string;
+
+    constructor(dockerPath: string){
+        this.dockerPath = dockerPath;
+    };
+
+    public pullImage(args: string[], silent?: boolean) {

```

Comment:
- Rename `pullImage` -> `pull` and `inspectImage` -> `inspect`?
- Should these methods have signature like `pull(image: string, args: string[], silent?: boolean)` as `image` is always required?

Reply:
done